GPy/models/sparse_gp_classification.py

```diff
@@ -40,6 +41,27 @@ def __init__(self, X, Y=None, likelihood=None, kernel=None, Z=None, num_inducing
 
         SparseGP.__init__(self, X, Y, Z, kernel, likelihood, inference_method=EPDTC(), name='SparseGPClassification',Y_metadata=Y_metadata)
 
+    @staticmethod
+    def from_sparse_gp(sparse_gp):
```

Comment:
Could this method have a more generic method name?

Add docstring.

Add a testcase.